WcaOnRails/db/migrate/20160811013347_add_announced_at_to_competitions.rb

```diff
@@ -0,0 +1,16 @@
+class AddAnnouncedAtToCompetitions < ActiveRecord::Migration
+  def up
+    add_column :Competitions, :announced_at, :datetime
+    execute <<-SQL
+      UPDATE Competitions
+      SET announced_at = (SELECT created_at
+        FROM posts
+        WHERE posts.title LIKE CONCAT(Competitions.name,'%')
```

Comment:
Actually not, because announcement titles start with the competition name, and results posts go like 'Jonatan Klosko wins Competition Name 2016'.
Thanks @Luis-J-Ianez for the smart query :)